### ISSUE TITLE
Commits and changesets information right below the corresponding work items details

### ISSUE THREAD

**Issue by edsonnarantes** [rfennell/AzurePipelines#402]
### Azure DevOps Extension you are using
Generate Release Notes Build Task (power shell)

### Where are you running it?
- Azure DevOps Service (VSTS)

### Version of Extension/Task
Public version

### Expected behavior and actual behavior
Is there a way to put commits and changesets information right below the corresponding work item details? There are two different loops with two different information running on them and I wanted to put this information together, first the work item and then the corresponding commit. 

### Steps to reproduce the problem
Edit the power shell template to organize work items information alongside with commit information.

**Comment by rfennell** [comment]
Unfortunately as you surmised they are two different loops, so you can't mix wi and commits

**Comment by edsonnarantes** [comment]
So, there's no way to implement, for example, a global variable (an array) that saves each wi detail information (running inside the wi loop), and then use this same array in the cs loop?

**Comment by rfennell** [comment]
Yes awkward to do, also I aiming to do no more major work on this extension, focusing on the cross platform version. Downside is that uses release Apis so removes the in build use case.

Before you look at has the same limitation
